system/jlib/jstats.cpp

```diff
@@ -983,6 +983,7 @@ static const constexpr StatisticMeta statsMetaData[StMax] = {
     { NUMSTAT(MatchRightRowsMax), "The largest number of right rows in a join group" },
     { NUMSTAT(MatchCandidates), "The number of candidate combinations of left and right rows forming join groups" },
     { NUMSTAT(MatchCandidatesMax), "The largest number of candidate combinations of left and right rows in a single group" },
+    { NUMSTAT(Threads), "The number of threads (strands) used by activity" },
```

Comment:
not necessarily strands in other contexts, e.g. if we start reporting number of threads used by keyed join, hash distrib.. Probably best to just drop "(strands)" here.